src/components/ai-edition/v4/EditorShellV4.module.css

```
@@ -406,6 +465,113 @@
+	}
+	.modeSwitch button {
+		padding: 4px 10px;
+	}
+}
+
+@media (max-width: 1080px) {
+	.topbar {
+		--topbar-gap: 8px;
+		--topbar-icon: 30px;
+		--topbar-project-w: 100px;
+		padding-left: calc(12px + var(--titlebar-inset-left, 0px));
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)